examples/config.sample

```diff
@@ -31,6 +31,11 @@
 #       Path to directory where to store flightstack log.
 #       No default value. If absent, no flightstack log will be stored.
 #
+#   LogMode
+#       One of <always>, <arm_until_disarm>
+#       Default: always. If absent, log from start until mavlink-router is
+#       stopped.
```

Comment:
also document the other one, mentioning the log file will be open when the vehicle is armed and closed when disarmed, creating a new log file.